### PR TITLE
Issue #6026: Highlight egg colors in context menu

### LINKED ISSUE #6026: A "Highlight egg colors" option in the Barbarian Assault plugin
Currently it is impossible to recolour 2> ground item names which have the same GP value. The only compromise is to highlight item1, default colour for item2, and then recolour a *menu* for item3 (requires a right-click). This is especially noticeable in Barbarian Assault, where the red, blue, and green eggs all have the same GP value. (there is also the yellow egg, on the last wave)

It would benefit a lot of players, especially those in the BA community, if in the "Barbarian Assault" plugin, there would be added a new toggle option such as "Highlight egg colors."

Hopefully that would isolate the need of otherwise having to make the ground item plugin even more complex and bloated.

Thanks for reading!

### LINKED ISSUE #6206: Add "Requires Battered key" to Elemental Workshop clues
image

### PR DESCRIPTION
**Pull request by Nava2**
* Highlights eggs with colour of egg

Color highlights

Closes #6026.

### CODE REVIEW

**Comment by Spedwards** [comment]
Closes #6026, not 6206.

File: runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java [hunk 1]
```diff
@@ -205,6 +254,11 @@ private void announceTime(String preText, String time)
 			.build());
 	}
 
+	private boolean isInGame()
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java]: What is the point of this method?

**Comment by Nava2** [comment]
> Closes #6026, not 6206.

Thanks! Fixed

File: runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java (hunk 1 above)
Comment (Nava2) [reply runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java]: Adjusted it a bit now and reused it within the plugin. Just checks that the current player is in game. The logic within the plugin is a bit confusing -- going to try and improve it and update the logic more.

**Pull request closed without merging**